apps/mosaico/src/Mosaico/Header/Menu.purs

```diff
@@ -116,103 +109,17 @@ render { props: { visible } } = DOM.div
-                       , { title: "Norden och världen"
-                         , url: ""
-                         }
-                       , { title: "Ekonomi"
-                         , url: ""
-                         }
-                       , { title: "Miljö och natur"
-                         , url: ""
-                         }
-                       ]
-                 }
-    opinion    = Section $
-                 { title: "OPINION"
-                 , modifier: Just "--section2"
-                 , url: ""
-                 , subsections:
-                     [ { title: "Ledare"
-                       , url: ""
-                       }
-                     , { title: "Kolumner"
-                       , url: ""
-                       }
-                     , { title: "Debatt"
-                       , url: ""
-                       }
-                     , { title: "Krönikor"
-                       , url: ""
-                       }
-                     ]
-                 }
-    kultur    =  Section $
-                 { title: "KULTUR"
-                 , modifier: Just "--section2"
-                 , url: ""
-                 , subsections:
-                     [ { title: "Litteratur"
-                       , url: ""
-                       }
-                     , { title: "Musik"
-                       , url: ""
-                       }
-                     , { title: "Scen"
-                       , url: ""
-                       }
-                     , { title: "Konst"
-                       , url: ""
-                       }
-                     , { title: "Film och TV"
-                       , url: ""
-                       }
-                     ]
-                 }
-    sport     =  Section $
-                 { title: "SPORT"
-                 , modifier: Just "--section3"
-                 , url: ""
-                 , subsections:
-                     [ { title: "Sport"
-                       , url: ""
-                       }
-                     , { title: "Handboll"
-                       , url: ""
-                       }
-                     , { title: "Ishockey"
-                       , url: ""
-                       }
-                     , { title: "Fotboll"
-                       , url: ""
-                       }
-                     , { title: "Motorsport"
-                       , url: ""
-                       }
-                     , { title: "Friidrott"
-                       , url: ""
-                       }
-                     , { title: "Skidsport"
-                       , url: ""
-                       }
-                     ]
-                 }
+    mkSection (n /\ acc) (Category c) =
+      let mkSubsection (Category subc) =
+            { title: subc.label, url: "/" <> show subc.label }
+          section =
+            Section $
+              { title: toUpper $ unwrap c.label
+              , modifier: Just $ "--section" <> show n
```

Comment:
Not sure if the `--section<n>` is used anywhere at this point for the categories, but maybe better to leave in if required by the `Section` type.

Reply:
I'm pretty sure it's a remnant of a previous version of the menu. I don't see any modifier mentioned in the `menu.less` file for sections.

Also I think that reading the category structure from the server makes it impossible to have ad hoc modifiers without coupling the server and client implementations, in the sens that the server would become aware of the layout that the client wants to render. All this to say that maybe it makes sense to remove that `modifier` field from the data structure :rofl: 
Unless we come up with some non ad hoc modifier we might want to attach on the fly but I can't come up with such an example now.

Reply:
Let's remove it 😁